### PR TITLE
delegation tests with integration of Api3Voting

### LINKED ISSUE #128: Add integration tests for api3-voting delegation
Currently delegation is checked just in the means of Unit tests, but delegation is used together with the api3-voting contract, so we need to check the work of this two modules with integration

### PR DESCRIPTION
**Pull request by BritikovKI**
Closes #128

### CODE REVIEW

File: packages/api3-voting/test/delegation-integration.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+        it('need to delegate', async () => {
+            await pool.delegateVotingPower(voter2, {from: voter1});
+            const neededSupport = pct16(60);
+            const minimumAcceptanceQuorum = pct16(20);
+            await voting.initialize(pool.address, neededSupport, minimumAcceptanceQuorum, votingDuration);
+            const voteId = createdVoteId(await voting.newVote(EMPTY_CALLS_SCRIPT, 'metadata', {from: voter3}));
+            await voting.vote(voteId, true, false, { from: voter2 });
+            const result = await voting.getVote(voteId);
+            expect(Number(result.yea)).to.equal(70);
```
Comment (kinspotting) [inline packages/api3-voting/test/delegation-integration.js:84]: no magic values pls

File: packages/api3-voting/test/delegation-integration.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        });
+
+        it('undo delegate', async () => {
+            const latest = Number(await time.latest());
+            await time.increaseTo(latest+Number(time.duration.weeks(1)));
+            await pool.undelegateVotingPower({from: voter1});
+            const voteId = createdVoteId(await voting.newVote(EMPTY_CALLS_SCRIPT, 'metadata', {from: voter3}));
+            await voting.vote(voteId, true, false, { from: voter2 });
+            const result = await voting.getVote(voteId);
+            expect(Number(result.yea)).to.equal(50);
```
Comment (kinspotting) [inline packages/api3-voting/test/delegation-integration.js:108]: same

File: packages/api3-voting/test/delegation-integration.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+        });
+
+        it('delegate delegated', async () => {
+            let latest = Number(await time.latest());
+            await time.increaseTo(latest+Number(time.duration.weeks(1)));
+            await pool.delegateVotingPower(voter2, {from: voter1});
+            await pool.delegateVotingPower(voter3, {from: voter2});
+            const voteId = createdVoteId(await voting.newVote(EMPTY_CALLS_SCRIPT, 'metadata', {from: voter3}));
+            const result = await voting.getVote(voteId);
+            expect(Number(result.yea)).to.equal(50);
```
Comment (kinspotting) [inline packages/api3-voting/test/delegation-integration.js:118]: same

**Review by kinspotting — CHANGES_REQUESTED** [review]
Needs minor changes

File: packages/api3-voting/test/delegation-integration.js (hunk 1 above)
Comment (BritikovKI) [reply packages/api3-voting/test/delegation-integration.js]: done

File: packages/api3-voting/test/delegation-integration.js (hunk 2 above)
Comment (BritikovKI) [reply packages/api3-voting/test/delegation-integration.js]: done

File: packages/api3-voting/test/delegation-integration.js (hunk 3 above)
Comment (BritikovKI) [reply packages/api3-voting/test/delegation-integration.js]: done

**Review by kinspotting — APPROVED** [review]
lgtm

**Pull request merged**